packages/gluestick/generator/predefined/app.js

```diff
@@ -35,7 +35,36 @@ module.exports = (options: GeneratorOptions) => {
         };
         return JSON.stringify(entries, null, '  ');
       },
-    },
+    }, {
+      file: '.flowconfig',
```

Comment:
Can we write a test for this logic? So you have a fake `.flowconfig` file, you fake to generated some new apps and compare the previous flow file with the new one ?